device-types/LYNX Technik AG/RFR-5014.yml

```diff
@@ -4,7 +4,7 @@ model: RFR 5014
 slug: lynx-technik-ag-rfr-5014
 u_height: 2.0
 is_full_depth: true
-airflow: passive
+airflow: front-to-rear
```

Comment:
The RFR 5014 has passive cooling.

Reply:
my bad, updated the wrong device, gah..

Reply:
although there is air flow accoring to this https://www.lynx-technik.com/uploads/tx_fyproducts/pdf/LYNX_R_FR_5012_5014_Manual_V2.1.pdf (but it's just passive?)